### PR TITLE
Change FLUO_DEV to UNO_HOME. Add doc.

### LINKED ISSUE #129: FLUO_DEV?
Is `FLUO_DEV` meant to point to the location of this Github repo, or does it have a different meaning?

In fetch.sh and all the setup-*.sh files, I replaced 

```bash
source "$FLUO_DEV"/bin/impl/util.sh
```

with

```bash
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../.."
source "$DIR"/bin/impl/util.sh
```

which gets the location of the Github repo, as long as the scripts are not moved or symlinked. I also changed `FLUO_DEV` to `DIR` when referenced.  Then the scripts worked.

### PR DESCRIPTION
**Pull request by dhutchis**
Closes #129

### CODE REVIEW

File: bin/impl/print-env.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ if [[ -z "$1" || "$1" == "--vars" ]]; then
 fi
 
 if [[ -z "$1" || "$1" == "--paths" ]]; then
-  echo -n "export PATH=\"\$PATH:$FLUO_DEV/bin:$HADOOP_PREFIX/bin:$ZOOKEEPER_HOME/bin:$SPARK_HOME/bin:$ACCUMULO_HOME/bin:$FLUO_HOME/bin"
+  echo -n "export PATH=\"\$PATH:$UNO_DEV/bin:$HADOOP_PREFIX/bin:$ZOOKEEPER_HOME/bin:$SPARK_HOME/bin:$ACCUMULO_HOME/bin:$FLUO_HOME/bin"
```
Comment (keith-turner) [inline bin/impl/print-env.sh]: why is this `UNO_DEV`?

Comment (dhutchis) [inline bin/impl/print-env.sh]: Whoops

File: README.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ The `uno` command uses `conf/env.sh.example` for its default configuration which
 sufficient for most users.
 
 Optionally, you can customize this configuration by creating an `env.sh` file and modifying it for
-your environment:
+your environment. Inside this script the variable `UNO_HOME` defaults to the location of the
+cloned Uno repository. You may override this by defining UNO_HOME yourself, if need be.
```
Comment (mikewalch) [inline README.md]: I think `You may override this by defining UNO_HOME yourself, if need be.` should be removed.  Before this PR, it was suggested in `load-env.sh` that users can override UNO_HOME.  However, I don't think we should allow this as I don't think it's necessary and adds complexity to scripts.

File: bin/impl/load-env.sh [hunk 3]
```diff
@@ -25,12 +25,12 @@ impl="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 bin="$( cd -P "$( dirname "$impl" )" && pwd )"
 # Stop: Resolve Script Directory
 
-# Determine FLUO_DEV - Use env variable set by user. If none set, calculate using bin dir
```
Comment (mikewalch) [inline bin/impl/load-env.sh]: Goes with above comment.  Don't think users should be able to set env.
```bash
# Determine UNO_HOME
UNO_HOME="$( cd -P "${bin}"/.. && pwd )"
```

File: conf/env.sh.example [hunk 4]
```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env bash
+# Variable UNO_HOME already defined, either by user or uno location in load-env.sh
```
Comment (mikewalch) [inline conf/env.sh.example]: Nice comment.  Could say `Variable UNO_HOME is already defined in load-env.sh to root directory of this repo.`

File: README.md [hunk 5]
```diff
@@ -126,7 +127,7 @@ You can confirm that everything started by checking the monitoring pages below:
 
  * [Hadoop NameNode](http://localhost:50070/)
  * [Hadoop ResourceManager](http://localhost:8088/)
- * [Accumulo Monitor](http://localhost:50095/)
+ * [Accumulo Monitor](http://localhost:50095/)  (new default Monitor address in Accumulo 1.8 uses [port 9995](http://localhost:9995/))
```
Comment (mikewalch) [inline README.md]: Nice comment.

File: README.md (hunk 2 above)
Comment (keith-turner) [inline README.md]: I agree, should not introduce without a good use case.  Adding it has maintenance implications.  In Accumulo @ctubbsii has discovered that ACCUMULO_HOME is not well defined and problematic for downstream packaging in Fedora.  For something like allowing Uno config to be stored in other locations, there are more targeted solutions that make less assumptions than `UNO_HOME`.

Comment (keith-turner) [inline README.md]: To clarify my comment, I am not against the scripts using `UNO_HOME` internally.  Just concerned about suggesting users can set it in the README.

**Comment by dhutchis** [comment]
Okay, I will un-document the ability to define your own UNO_HOME, but leave the functionality to do so in.  Good point re maintenance.

**Comment by mikewalch** [comment]
@keith-turner, you can squash and merge on GittHub.  It should leave @dhutchis as author.  I will do it now.

**Pull request merged**